packages/scripts/data/locations.csv

```diff
@@ -0,0 +1,85 @@
+Health Authority,Health Service Delivery Areas,Local Health Areas

```

Comment:
Since the source of truth for will be an extract, maybe it would be good to gitignore both csvs